src/shared/logs-show.c

```diff
@@ -317,62 +317,49 @@ static bool print_multiline(
         return ellipsized;
 }
 
-static int output_timestamp_monotonic(FILE *f, sd_journal *j, const char *monotonic) {
-        sd_id128_t boot_id;
-        uint64_t t;
-        int r;
-
+static int output_timestamp_monotonic(FILE *f, dual_timestamp *ts) {
```

Comment:
should be const

Reply:
Changed here and on all the other places